### PR TITLE
Font styles

### PR DESCRIPTION
**Pull request by jamest1903**

### CODE REVIEW

File: background/background.mjs [hunk 1]
```diff
@@ -533,25 +533,26 @@ const sendDailyNotification = async (activeTabId) => {
...
+    weekStartMonday.setDate(now.getDate() - (now.getDay() + 6) % 7); // Calculate last Monday's date
   }
 
   if (activeTab[0].id) {
-    const weekDate =  new Date(notificationsStatus.lastDisplayedWeeklyTimeStamp);
-    const date = new Date(notificationsStatus.lastDisplayedDailyTimeStamp);
-    if (!weekDate.valueOf() || weekDate < lastMonday) {
+    const lastWeeklyDisplay =  new Date(notificationsStatus.lastDisplayedWeeklyTimeStamp);
+    const lastDailyDisplay = new Date(notificationsStatus.lastDisplayedDailyTimeStamp);
+    if (!lastWeeklyDisplay.valueOf() || lastWeeklyDisplay < weekStartMonday) {
```
Comment (slecorne) [inline background/background.mjs:551]: what happens if we are Tuesday at 17:00 and we showed the notification Monday at 9:00.
I have the impression that you would compare Mon 9:00 < Mon 17:00 and show the notification. Probably you need to set hours 00:00:00 for all cases

File: background/background.mjs [hunk 2]
```diff
@@ -533,25 +533,26 @@ const sendDailyNotification = async (activeTabId) => {
...
       sendWeeklyNotification(activeTab[0].id);
       return; // User to acknowledge popup before replacing with weekly below.
     };
-    if (!date.valueOf() || date.getDate() < now.getDate()) {
+    if (!lastDailyDisplay.valueOf() || lastDailyDisplay.getDate() < now.getDate()) {
```
Comment (slecorne) [inline background/background.mjs:555]: if you have a notification shown on a 31 of any month, you would never show anything again, no?

File: webpack/src/components/Popup.vue [hunk 3]
```diff
@@ -82,6 +82,15 @@ export default {
 
 
 <style scoped>
+/* dialog {
```
Comment (slecorne) [inline webpack/src/components/Popup.vue:85]: we can delete it I think

**Review by slecorne — CHANGES_REQUESTED** [review]

File: background/background.mjs (hunk 1 above)
Comment (jamest1903) [reply background/background.mjs:551]: let weekStartMonday = new Date(now);
  if (!todayIsMonday) {
    weekStartMonday.setDate(now.getDate() - (now.getDay() + 6) % 7); // Calculate last Monday's date
  }
  weekStartMonday.setHours(0,0,0);

**Review by slecorne — APPROVED** [review]

**Pull request merged**